src/renderers/shaders/UniformsLib.js

```diff
@@ -18,6 +18,7 @@ var UniformsLib = {
 		offsetRepeat: { value: new Vector4( 0, 0, 1, 1 ) },
 
 		specularMap: { value: null },
+		uSpecularStrength: { value : 1 },
```

Comment:
I'll let @bhouston and @WestLangley decide if this should go in or not. However... Would be nice if you followed the pattern and named this uniform `speculatStrength`.